build-publish.dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
18.04 is even smaller. Hesitant to go to 20.04 though, too new.

Reply:
Yeah perhaps we should give it a go, we've avoided moving off 16.04 but here it would probably be fine.